sandbox/interactivity-sandbox.js

```diff
@@ -161,10 +166,20 @@ function sandbox() {
 
 function save_current_code(code) {
     if (!localStorage) throw new Error('Local storage support required');
-    localStorage.setItem('_current_code', code);
+    localStorage.setItem('_i_current_code', code);
 }
 
 function load_last_code() {
     if (!localStorage) throw new Error('Local storage support required');
-    return localStorage.getItem('_current_code') || '';
+    return localStorage.getItem('_i_current_code') || '';
+}
+
+function save_current_json(json) {
+    if (!localStorage) throw new Error('Local storage support required');
```

Comment:
also some people can disable localStorage in browser settings. Also there's could be problems in some browsers in incognito mode.